### PR TITLE
New version: Geeks3D.FurMark version 1.38.1.0

### LINKED ISSUE #141730: [Package Issue]: Geeks3D.FurMark
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

No applicable update found.

### Brief description of your issue

Current Version in winget list is 1.37.2
There are already newer versions out there, latest is **1.38.1.0** and **2.1.0.2**
Which can be found here: https://geeks3d.com/furmark/downloads/

### Steps to reproduce

Use command: `winget search Geeks3D.FurMark`
Get result:
```
Name    Id              Version  Source
----------------------------------------
FurMark Geeks3D.FurMark 1.37.2.0 winget
```

### Actual behavior

winget lists older version of FurMark

### Expected behavior

Latest version of FurMark

### Environment

```raw
Windows Package Manager v1.7.10582
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.3155
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.22.10582.0

Winget Directories
-----------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag…
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett…
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages
Installer Downloads                %USERPROFILE%\Downloads

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting                             State
--------------------------------------------------
LocalManifestFiles                        Disabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride                     Disabled
LocalArchiveMalwareScanOverride           Disabled

Other:
WingetUI
```

### Screenshots and Logs

image

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.1.0 :rocket:

- Resolves #141730

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141766)

### CODE REVIEW

File: manifests/g/Geeks3D/FurMark/1.38.1.0/Geeks3D.FurMark.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Geeks3D.FurMark
+PackageVersion: 1.38.1.0
+InstallerType: inno
+ReleaseDate: 2024-02-15
+Installers:
+- Architecture: x86
+  InstallerUrl: https://geeks3d.com/dl/get/738
```
Comment (fancybody) [inline manifests/g/Geeks3D/FurMark/1.38.1.0/Geeks3D.FurMark.installer.yaml:10]: @Exorcism0666
image
```suggestion
  InstallerUrl: https://files1.majorgeeks.com/10afebdbffcd4742c81a3cb0f6ce4092156b4375/benchmark/FurMark_1.38.1.0_Setup.exe
```
[Discussion on previous version](https://github.com/microsoft/winget-pkgs/issues/137134#issuecomment-1918301420)

**Review by fancybody — CHANGES_REQUESTED** [review]
@Exorcism0666
image
[Discussion on previous version](https://github.com/microsoft/winget-pkgs/issues/137134#issuecomment-1918301420)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Hi @og-mrk,

Other community members have discussed various properties such as the ones you've quoted. These two properties are generally considered less important than other properties, and the consensus was to not block when they are missing.

**Comment by stephengillie** [comment]
Completing validation.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Reset approval workflow.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**